grpc-haskell.cabal

```diff
@@ -19,7 +19,7 @@ Flag Debug
 flag with-examples
   description: Also build example executables.
   manual:      True
-  default:     False
+  default:     True
```

Comment:
What is the reason for changing this? A user may not necessarily want examples built by-default and would have to know to turn them off explicitly if they don't care about it.